File: Lists in Java/Person.java

public class Person {
    private String name;
    private String address;
    private String phone;

    public Person(String newName, String address, String phone) {
        this.name = newName;
        this.address = address;
        this.phone = phone;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return this.name;
    }
    
    public String getPhone() {
        return this.phone;
    }

    @Override
    public String toString() {
        return "Name: " + name + "\nAddress: " + address + "\nPhone: " + phone;
    }
}

File: Linked Lists/Person.py
class Person:
    def __init__(self, newName="none", address="none", phone="999-999-9999"):
        self.name = newName
        self.address = address
        self.phone = phone

    def setName(self, name):
        self.name = name

    def getName(self):
        return(self.name)

    def getPhone(self):
        return(self.phone)
    
    def __str__(self):
        stringtoReturn = "Name         : " + self.name \
                       + "\naddress      : " + self.address \
                       + "\nphone        : " + self.phone 
        return(stringtoReturn)
